pkg/analyzer/proto.go

```diff
@@ -69,6 +77,69 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
+func makeExpr(expr *ast.SelectorExpr) (oldExpr, newExpr string, pos, end token.Pos) {
+	pos = expr.Pos()
+	end = expr.End()
+
+loop:
+	for {
+		var end2 token.Pos
+
+		switch x := expr.X.(type) {
+		case *ast.Ident:
+			if oldExpr != "" {
+				break loop
+			}
+
+			oldExpr = fmt.Sprintf("%s.%s", x.Name, expr.Sel.Name)
+			newExpr = fmt.Sprintf("%s.Get%s()", x.Name, expr.Sel.Name)
+			break loop
+
+		case *ast.SelectorExpr:
+			oldExpr, newExpr, _, end2 = makeExpr(x)
+			if end2 > end {
+				end = end2
+			}
+
+			if oldExpr == "" {
+				oldExpr = fmt.Sprint(x)
+				newExpr = fmt.Sprint(x)
+			}
+
+			oldExpr = fmt.Sprintf("%s.%s", oldExpr, expr.Sel.Name)
+			newExpr = fmt.Sprintf("%s.Get%s()", newExpr, expr.Sel.Name)
+
+			vv, ok := x.X.(*ast.SelectorExpr)
+			if !ok {
+				break loop
+			}
+			expr = vv
+
+		case *ast.CallExpr:
+			v, ok := x.Fun.(*ast.SelectorExpr)
+			if ok {
+				oldExpr, newExpr, _, end2 = makeExpr(v)
+				if end2 > end {
+					end = end2
+				}
+
+				oldExpr = strings.ReplaceAll(oldExpr, "GetGet", "Get")
```

Comment:
Not a perfect solution, but it works. Unfortunately, I don't have time to look for another solution, without copy-paste.